district42/types/_list_schema.py

```diff
@@ -45,11 +46,13 @@ def max_len(self) -> Nilable[int]:
 
 
 class ListSchema(Schema[ListProps]):
+    type: TypeAlias = List
+
     def __accept__(self, visitor: SchemaVisitor[ReturnType], **kwargs: Any) -> ReturnType:
         return visitor.visit_list(self, **kwargs)
 
     def __call__(self, /,
-                 elements_or_type: Union[List[ElementType], GenericSchema]) -> "ListSchema":
+                 elements_or_type: Union[List[ElementType], GenericSchema, Any]) -> "ListSchema":
```

Comment:
Could you please clarify the reason for adding `Any` to the `elements_or_type` type list? I'm curious to understand the intent behind this change. Thank you!

Reply:
Its solves typing error for lists

```
def random_services(...) -> dict[str, Any]:
    context_rubric = fake(schema.list(GeneralContextRubricSchema).len(1, ...))


helpers/generators/webapi/random_service.py:11: error: Argument 1 to "__call__" of "ListSchema" has incompatible type "type[GeneralContextRubricSchema]"; expected "list[Schema[Any] | EllipsisType] | Schema[Any]"  [arg-type]
```
I could not fount another way to solve it, cause ListSchema.__call__ doesn't know about any custome types generated by d42-typing